docs/guided-tour.md

```diff
@@ -88,15 +104,16 @@ Sandstorm supports a number of publishing apps, including WordPress and Ghost. (
 
 Try them out!
 
-* Click the **Sandstorm** logo in the top-left, and
-* Click **Install apps** and
-* Find _WordPress_ or _Ghost_ on the page, and click **Install*.
-* Click **New WordPress site** (or similar for Ghost).
+* Click **Apps** in the left sidebar
+* Click **Install...** to go to the App Market
+* Find _WordPress_ or _Ghost_, and click **Install*.
+* Confirm your **Install**
+* Click **Create new site** (or similar for Ghost).
 
 This takes you to WordPress running on your server. You are automatically logged-in and can start writing. You can click **Rebuild public site** and this snapshots your blog as static HTML and exports it at a random hostname.
 
 You can also configure a custom domain to map to the same static content. Only people with whom you **share** the WordPress instance can interact with WordPress's PHP code. The result is that WordPress becomes a security-hardened collaborative static site generator.
```

Comment:
FWIW, what I meant was in these paragraphs:

"This takes you to WordPress running on your server..."

and

"You can also configure a custom domain to map to the same static content..."

the paragraph refers to WordPress, but the user may have chosen the Ghost route, in which case, this sentence won't be true: "This takes you to WordPress" - it might take them to Ghost.

This issue is present in the existing text, so it's not like you need to be the one to fix it, but it struck me, so I thought I'd mention it.